lib/internal/fs/utils.js

```diff
@@ -667,15 +668,11 @@ const defaultRmdirOptions = {
 const validateRmOptions = hideStackFrames((path, options, callback) => {
   try {
     options = validateRmdirOptions(options, defaultRmOptions);
+    validateBoolean(options.force, 'force');
```

Comment:
Should we change the name to `options.force` (and similar in other places) while we are at it? (`validateBoolean(options.force, 'options.force');`)
This will make it clearer for the user and also consistent with other places of handling `options` object validation.

Reply:
@lundibundi @cjihrig this seems like a good recommendation to me, do we do this elsewhere with option validation?

Reply:
> do we do this elsewhere with option validation?

Like most things in core, it's implemented inconsistently. I think it's a good change though. I'll update this PR.

Reply:
OK, updated.